.github/workflows/CI_build_combined.yml

```diff
@@ -0,0 +1,208 @@
+
+    - name: build cmake libs
+      run: |
+           cmake --build ${{ env.BUILD_DIR_LIBS }} --config ${{ matrix.build_configuration }}
+           cmake --install ${{ env.BUILD_DIR_LIBS }}
+
+    # - name: generate cmake
+      # run: |
+           # cmake -G "${{ matrix.build_platform }}" -DCMAKE_BUILD_TYPE="${{ matrix.build_configuration }}" -DCMAKE_PREFIX_PATH=./_build_libs/dist -DCMAKE_TOOLCHAIN_FILE=../cmake/android.toolchain.cmake -B ${{ env.BUILD_DIR_APP }}
+
+    # - name: build cmake
+      # run: |
+           # cmake --build ${{ env.BUILD_DIR_APP }} --config ${{ matrix.build_configuration }}
+           # cmake --install ${{ env.BUILD_DIR_APP }}
+
+    # - name: run ctest
+      # run: |
+           # ctest --test-dir ${{ env.BUILD_DIR_APP }} --output-on-failure -C "${{ matrix.build_configuration }}"
+
+  build_macos:
+
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_configuration: [Release]
+        build_platform: ["Ninja"]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: install ninja
+      run: |
+           brew install ninja
+
+    - name: generate cmake libs
+      run: |
+           cmake -G "${{ matrix.build_platform }}" -DCMAKE_BUILD_TYPE="${{ matrix.build_configuration }}" -B ${{ env.BUILD_DIR_LIBS }}  ./libs
+
+    - name: build cmake libs
+      run: |
+           cmake --build ${{ env.BUILD_DIR_LIBS }} --config ${{ matrix.build_configuration }}
+
+    - name: generate cmake
+      run: |
+           cmake -G "${{ matrix.build_platform }}" -DCMAKE_BUILD_TYPE="${{ matrix.build_configuration }}" -DCMAKE_PREFIX_PATH=./_build_libs/dist -B ${{ env.BUILD_DIR_APP }}
+
+    # due to https://bugreports.qt.io/browse/QTBUG-117765 QT 6.5.3 fails
+    # - name: build cmake
+      # run: |
+           # cmake --build ${{ env.BUILD_DIR_APP }} --config ${{ matrix.build_configuration }}
+           # cmake --install ${{ env.BUILD_DIR_APP }}
+
+    # - name: run ctest
+      # run: |
+           # ctest --test-dir ${{ env.BUILD_DIR_APP }} --output-on-failure -C "${{ matrix.build_configuration }}"
+
+
+  build_ios:
+
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_configuration: [Release]
+        build_platform: ["Unix Makefiles"]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: install ninja
+      run: |
+           brew install ninja
+
+    - name: generate cmake libs
+      run: |
+           cmake -G "${{ matrix.build_platform }}" -DCMAKE_BUILD_TYPE="${{ matrix.build_configuration }}" -DCMAKE_TOOLCHAIN_FILE=../cmake/iOS.toolchain.cmake -B ${{ env.BUILD_DIR_LIBS }}  ./libs
+
+    - name: build cmake libs
+      run: |
+           cmake --build ${{ env.BUILD_DIR_LIBS }} --config ${{ matrix.build_configuration }}
+
+    # - name: generate cmake
```

Comment:
remove comment please... let's create a PR after this

Reply:
@misery Did I get you right that the known failing parts should be active to see the problems?